conf/webknossos.latest.routes

```diff
@@ -250,9 +250,9 @@ GET           /termsOfService/acceptanceNeeded
 
 # Time Tracking
 # Note, there is also /users/:id/loggedTime
-GET           /time/user/:userId/spans                                                          controllers.TimeController.timeSpansOfUser(userId: String, start: Long, end: Long, annotationTypes: String, projectIds: Option[String])
-GET           /time/user/:userId/summedByAnnotation                                             controllers.TimeController.timeSummedByAnnotationForUser(userId: String, start: Long, end: Long, annotationTypes: String, projectIds: Option[String])
-GET           /time/overview                                                                    controllers.TimeController.timeOverview(start: Long, end: Long, annotationTypes: String, teamIds: Option[String], projectIds: Option[String])
+GET           /time/user/:userId/spans                                                          controllers.TimeController.timeSpansOfUser(userId: String, start: Long, end: Long, annotationTypes: String, projectIds: Option[String], annotationStates: String)
+GET           /time/user/:userId/summedByAnnotation                                             controllers.TimeController.timeSummedByAnnotationForUser(userId: String, start: Long, end: Long, annotationTypes: String, projectIds: Option[String], annotationStates: String)
+GET           /time/overview                                                                    controllers.TimeController.timeOverview(start: Long, end: Long, annotationTypes: String, teamIds: Option[String], projectIds: Option[String], annotationStates: String)
```

Comment:
I think the order isn’t relevant here for the compiler, as this is, so to speak, always called by name. If an optional parameter is omitted from a request query, it is auto-filled with None (different from Some(null). In fact we want to avoid null wherever possible, since the type-checker does not deal with it).

However, you are right; at least for style, I’ll move the new parameter so that the optional ones stay last.